packages/caravan-fees/src/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from "./types";
+export * from "./utils";
+export * from "./transactionAnalyzer";
+export * from "./rbf";
+export * from "./cpfp";
+export * from "./constants";
```

Comment:
Needed to `export * from "./btcTransactionTemplate";` to get this building

Reply:
thanks for the catch, corrected this now ... added export for btcTransactionComponents too